### ISSUE TITLE
Redis export: either no data or "ERR syntax error"

### ISSUE THREAD

**Issue by jordanade** [TablePlus/TablePlus#2309]
Redis 5.0.6. TablePlus 3.12.2 (358).

Try to export Redis DB. Export to SQL produces empty file (just has comment on top). Export to CSV produces "value\nERR syntax error". Export to JSON produces same error but in JSON format.

**Comment by brettflorio** [comment]
I'm running into this as well. Two databases, different data, same errors as @jordanade. (Same TablePlus version as well. Redis 6.0.10.)

**Comment by jordanade** [comment]
Yes, I am using Elasticache as well, over SSH. Still happening with Redis 6.0.5 and TablePlus 3.12.5.

**Comment by zappzerapp** [comment]
I'm also affected 😞
